src/content.rs

```
@@ -56,15 +58,22 @@ impl Content {
             editing: SecondaryMap::new(),
             task_input_ids: SecondaryMap::new(),
             input: String::new(),
+            hide_complete: false,
         }
     }
 
     fn list_header<'a>(&'a self, list: &'a List) -> Element<'a, Message> {
         let spacing = theme::active().cosmic().spacing;
         let export_button = widget::button::icon(icons::get_handle("share-symbolic", 18))
+            .class(cosmic::style::Button::Suggested)
+            .padding(spacing.space_xxs)
+            .on_press(Message::ToggleComplete);
+
+        let hide_complete_button = widget::button::icon(icons::get_handle("share-symbolic", 18))
```

You're using the same icon that's being used for the share button.

---

sorry, i forgot to mention it, I am not sure what icon should be used. I looked at cosmic-icons repository and couldn't find any that would suit. Maybe this shouldn't be a icon button at all, maybe i could move it view menu?

---

sorry, i am new to rust and iced. If i understand correctly there should be flag in application state that would be updated from view and also possible persisted in some config/state, and flag on content state that would work on the list it has?

---

To clarify my previous request, I'm proposing a system with two levels of control for managing the visibility of completed tasks:

- **Application**: I envision a menu item, specifically a checkbox, that governs a global configuration property. When this checkbox is enabled, the application would, by default, hide completed tasks in all task lists. This global preference would be persisted within the application's configuration files.

- **List-Specific**: In addition to the global setting, each individual task list should feature its own toggle (similar to the one you previously implemented). This toggle would manage a list-specific setting for the visibility of completed tasks. This per-list setting would be saved within the configuration of that particular list.

---

Regardless of that, I think we can proceed with this PR and add the global config later on.

Please find an icon with an eye to symbolize what the button is doing and we can proceed with the merge.